src/main.c

```diff
@@ -81,6 +81,8 @@ static void usage(int code, int keep_it_short) {
       "  -M               monochrome (don't colorize JSON);\n"
       "  -S               sort keys of objects on output;\n"
       "  --tab            use tabs for indentation;\n"
+      "  --indent         use indent amount of spaces for indentation.\n"
```

Comment:
Maybe should be `--indent n` to be clearer?

Reply:
@wader fixed

Reply:
👍 hopefully will get merged eventually, see #2305 for details